Doc/library/logging.rst

```diff
@@ -1183,6 +1183,14 @@ functions.
    |              | with 'filename' or 'stream' - if both are   |
    |              | present, a ``ValueError`` is raised.        |
    +--------------+---------------------------------------------+
+   | ``force``    | If this keyword argument is specified as    |
+   |              | true, basicConfig removes all handlers of   |
```

Comment:
... is specified as true, any existing handlers attached to the root logger are removed and closed, before carrying out the configuration as specified by the other arguments.